doc/reference/peripherals/espi.rst

```diff
@@ -0,0 +1,13 @@
+.. _espi_api:
+
+eSPI
+####
+
+Overview
+********
+
```

Comment:
Is it possible to add a short overview? Having an empty section doesn't make much sense

Reply:
Good catch @gmarull.

Please add something minimal in the Overview section @albertofloyd.

Reply:
Added brief intro on this bus and link to full spec